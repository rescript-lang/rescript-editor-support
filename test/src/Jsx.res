module M = {
  @react.component
  let make = (~first, ~fun="", ~second="") => React.string(first ++ fun++ second)
}

let d = <M first="abc" />
//       ^def

//^com <M second=fi 


//^com <M second="abc" f
