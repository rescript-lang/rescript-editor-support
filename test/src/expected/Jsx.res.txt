{"uri": "Jsx.res", "range": "{"start": "{"line": "2", "character": "6"}", "end": "{"line": "2", "character": "10"}"}"}
[]
[{
    "label": "key",
    "kind": 4,
    "tags": [],
    "detail": "string",
    "documentation": {"kind": "markdown", "value": "\nJsx.res:11"}
  }, {
    "label": "first",
    "kind": 4,
    "tags": [],
    "detail": "string",
    "documentation": {"kind": "markdown", "value": "\nJsx.res:11"}
  }, {
    "label": "fun",
    "kind": 4,
    "tags": [],
    "detail": "option<string>",
    "documentation": {"kind": "markdown", "value": "\nJsx.res:11"}
  }]
